src/visualizations/config/adapters/dhis_highcharts/chart.js

```diff
@@ -31,17 +34,30 @@ const getEvents = () => ({
                     })
                 }
             })
+            renderCustomSVG.call(this)
         },
     },
 })
 
-export default function (layout, el, dashboard) {
+export default function (layout, el, extraOptions, series) {
     return Object.assign(
         {},
         getType(layout.type),
         { renderTo: el || layout.el },
         DEFAULT_CHART,
-        dashboard ? DASHBOARD_CHART : undefined,
-        getEvents()
+        extraOptions.dashboard ? DASHBOARD_CHART : undefined,
+        getEvents(),
+        layout.type === VIS_TYPE_SINGLE_VALUE
+            ? {
+                  backgroundColor: getSingleValueBackgroundColor(
+                      extraOptions.legendOptions,
+                      extraOptions.legendSets,
+                      series[0]
+                  ),
```

Comment:
In Highcharts it is possible to define a background color for a chart. As such, I thought it needed to be set here, instead of being updated later by the custom visualisation code.